doc/source/getting_started/contributor_resources.rst

```diff
@@ -0,0 +1,12 @@
+Contributor Resources
+=====================
+.. _contributor_resources:
+
+Summary:
+
+.. toctree:: 
+    :maxdepth: 2
+
+    ../../../LICENSE
+    ../../../DEPENDENCIES
```

Comment:
I'm not sure why but when I look at the generated docs this displays the requirements.txt file which is used for testing dev versions of some dependencies. Was the intention is to show all dependencies? I'm not sure what a good way to do this is pulling from `pyproject` or `setup`.
That said I think this page needs to show what a potential contributor should do. There are similar pages on jwst or other packages.